l10n/en-US/viewer.properties

```diff
@@ -43,7 +43,9 @@ save.title=Save
 save_label=Save
 bookmark1.title=Current Page (View URL from Current Page)
 bookmark1_label=Current Page
+# LOCALIZATION NOTE (open_in_app.title): This string is used in Firefox for Android. Length of the translation matters since we are in a mobile context, with limited screen estate.
```

Comment:
Is this true? I assume this would be used as a tooltip, so there are no constraints (assuming a tooltip is even displayed on mobile)

Reply:
I'm not sure how it shows up tbh, but as it is only for Android I thought it might be better to have it here too.
@calixteman can explain how the title is used when he's back on Monday

Reply:
I think it's used to set [the title here](https://github.com/mozilla/pdf.js/blob/356f3cf2856db4113c4489c0398ac7b32cbc93c5/web/viewer-geckoview.html#L87). This can wait next week, as I've already added a comment in Pontoon in the meantime.

Reply:
Perfect, thanks!

Reply:
On mobile, `title` should be ignored in general:
https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute
but I added it in case it's useful for accessibility.
So I'd say that we don't care that much about the size.

Reply:
All right, removed it!